ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -114,7 +114,61 @@ import RevenueCat
         // See https://sdk.revenuecat.com/android/5.1.1/purchases/com.revenuecat.purchases/-purchases/-companion/can-make-payments.html
         return Purchases.canMakePayments()
     }
+}
+
+// MARK: Refund request
+@objc public extension CommonFunctionality {
+#if os(iOS)
+    @available(iOS 15.0, *)
+    @available(tvOS, unavailable)
+    @available(macOS, unavailable)
+    @available(watchOS, unavailable)
+    @available(macCatalyst, unavailable)
+    @objc(beginRefundRequestProductId:completionBlock:)
+    static func beginRefundRequest(productId: String,
+                                   completion: @escaping (ErrorContainer?) -> Void) {
```

Comment:
In iOS, the result is an enum with 3 values `valid`, `userCancelled` and `error`. So to keep it simple, here I'm just returning an optional `ErrorContainer` which would indicate if it was `userCancelled` or another error, and if it's missing, it would mean it was successful. Alternatively, I could map it to another enum from PHC which can be accessed by the hybrids. Thoughts on that?